### PR TITLE
rabbitmq-c 0.7.0

### PR DESCRIPTION
**Pull request by guoxiao**
update `rabbitmq-c` depends
* `rabbitmq-c` does not depend on `rabbitmq` server.
* `rabbitmq-c` depends on `openssl`

update to version 0.7.0

### CODE REVIEW

File: Library/Formula/rabbitmq-c.rb [hunk 1]
```diff
@@ -31,10 +29,10 @@ def install
     args << "-DBUILD_API_DOCS=OFF"
 
     args << if build.with? "tools"
-      "-DBUILD_TOOLS=ON"
-    else
-      "-DBUILD_TOOLS=OFF"
-    end
+              "-DBUILD_TOOLS=ON"
```
Comment (DomT4) [inline Library/Formula/rabbitmq-c.rb]: Indentation has slipped way off here. Suspect you've indented with two tabs instead of two spaces.

Comment (guoxiao) [inline Library/Formula/rabbitmq-c.rb]: It's not a mistake. I aligned else with if to fix `brew audit --strict` warning.

**Comment by mikemcquaid** [comment]
`simple-amqp-client` (and perhaps other `brew uses rabbitmq-c` things) need a `revision` added or increased.

**Comment by guoxiao** [comment]
Some php extensions should be updated to be compatible with updated `rabbitmq-c`.
see homebrew/homebrew-php#1806

**Comment by DomT4** [comment]
Is this good to :ship: or?

**Comment by mikemcquaid** [comment]
Yup :+1:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/92c66867f2914bcf50df3ad799573a89e09dc8c9. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:.

If you could squash your commits on final push in future that'd be appreciated. We try to keep the commit tree relatively clean and squashing locally isn't always easy when you aren't the person who wrote the commits. Thanks!

**Pull request closed without merging**